lib/CL/devices/cuda/pocl-cuda.c

```diff
@@ -802,6 +802,34 @@ pocl_cuda_free (cl_device_id device, cl_mem mem_obj)
   p->version = 0;
 }
 
+int
+pocl_cuda_can_migrate_d2d (cl_device_id dest, cl_device_id source)
+{
+  assert (dest != source);
+  if (strcmp (source->ops->device_name, source->ops->device_name) == 0)
```

Comment:
Just for my understanding, does this mean you can't migrate from two CUDA devices with different names?

Reply:
And while replying to this I noticed I had copied & pasted "source" there twice. :laughing:  Fixed.